cypress/integration/settings/create_new_parzelle_spec.js

```diff
@@ -1,15 +1,18 @@
 describe('Create new Parzelle/Allotment', function() {
-  before(function() {
-    cy.visit('/window/540210');
+  let name;
+  it('Read fixture and prepare the names', function() {
+    cy.fixture('settings/create_new_parzelle_spec.json').then(f => {
+      name = f['name'];
+    });
   });
-
-  it('open new parzelle', function() {
+  it('Open new parzelle', function() {
+    cy.visit('/window/540210');
```

Comment:
can we please replace this with `cy.visitWindow(540210, 'NEW');` ?

Reply:
done.